po/fr.po

```diff
@@ -235,347 +265,347 @@ msgstr ""
 
 #, c-format
 msgid "%s: %s: expected %d arguments; got %d\n"
-msgstr ""
+msgstr "%s : %s : %d arguments attendus ; %d reçus\n"
 
 #, c-format
 msgid "%s: %s: invalid base value\n"
-msgstr ""
+msgstr "%s : %s : valeur de base invalide\n"
 
 #, c-format
 msgid "%s: %s: invalid function name"
-msgstr ""
+msgstr "%s : %s : nom de fonction invalide"
 
 #, c-format
 msgid "%s: %s: invalid integer\n"
-msgstr ""
+msgstr "%s : %s : entier invalide\n"
 
 #, c-format
 msgid "%s: %s: invalid mode\n"
-msgstr ""
+msgstr "%s : %s : mode invalide\n"
 
 #, c-format
 msgid "%s: %s: invalid mode name. See `help identifiers`\n"
-msgstr ""
+msgstr "%s : %s : nom de mode invalide. Voir `help identifiers`\n"
 
 #, c-format
 msgid "%s: %s: invalid scale\n"
-msgstr ""
+msgstr "%s : %s : échelle invalide\n"
 
 #, c-format
 msgid "%s: %s: invalid subcommand\n"
-msgstr ""
+msgstr "%s : %s : sous-commande invalide\n"
 
 #, c-format
 msgid "%s: %s: invalid variable name. See `help identifiers`\n"
-msgstr ""
+msgstr "%s : %s : nom de variable invalide. Voir `help identifiers`\n"
 
 #, c-format
 msgid "%s: %s: option does not take an argument\n"
-msgstr ""
+msgstr "%s : %s : cette option ne prend pas d’argument\n"
 
 #, c-format
 msgid "%s: %s: option requires an argument\n"
-msgstr ""
+msgstr "%s : %s : cette option doit être utilisé avec un argument\n"
 
 #, c-format
 msgid "%s: %s: subcommand takes no options\n"
-msgstr ""
+msgstr "%s : %s : cette sous-command n’a pas d’option\n"
 
 #, c-format
 msgid "%s: %s: unexpected positional argument"
-msgstr ""
+msgstr "%s : %s : argument positionnel inattendu"
 
 #, c-format
 msgid "%s: %s: unknown option\n"
-msgstr ""
+msgstr "%s : %s : option inconnue\n"
 
 #, c-format
 msgid "%s: '%s' is a broken symbolic link to '%s'\n"
-msgstr ""
+msgstr "%s : « %s » est un lien symbolique cassé vers « %s »\n"
 
 #, c-format
 msgid "%s: '%s' is not a directory\n"
-msgstr "%s : '%s' n’est pas un dossier\n"
+msgstr "%s : « %s » n’est pas un dossier\n"
 
 #, c-format
 msgid "%s: '%s' is not a job\n"
-msgstr "%s : '%s' n’est pas une tâche\n"
+msgstr "%s : « %s » n’est pas une tâche\n"
 
 #, c-format
 msgid "%s: '%s' is not a valid job ID\n"
-msgstr ""
+msgstr "%s : « %s » n’est pas un ID de tâche valide\n"
 
 #, c-format
 msgid "%s: '%s' is not a valid process ID\n"
-msgstr "%s : '%s' n'est pas un PID valide\n"
+msgstr "%s : « %s » n’est pas un ID de processus valide\n"
 
 #, c-format
 msgid "%s: --command cannot be combined with --position=command"
-msgstr ""
+msgstr "%s : --command ne peut pas être utilisé avec l’option --position"
 
 #, c-format
 msgid "%s: --function option requires --add\n"
-msgstr ""
+msgstr "%s : l’option --function doit être utilisé avec --add\n"
 
 #, c-format
 msgid "%s: --position option requires --add\n"
-msgstr ""
+msgstr "%s : l’option --position doit être utilisé avec --add\n"
 
 #, c-format
 msgid "%s: --regex option requires --add\n"
-msgstr ""
+msgstr "%s : l’option --regex doit être utilisé avec --add\n"
```

Comment:
```suggestion
msgstr "%s : l’option --regex doit être utilisée avec --add\n"
```

And the rest as I don't want to spam with messages... Maybe we should also do that when it says "%s: --something doit être utilisé" as "option" is implied. :shrug: